aws_s3_bucket.tf

```diff
@@ -121,3 +149,254 @@ resource "aws_s3_bucket" "s3MFADeleteOnly" {
     mfa_delete = true
   }
 }
+
+resource "aws_s3_bucket_policy" "s3VersioningMfaFalsePolicy" {
+  bucket = "${aws_s3_bucket.s3VersioningMfaFalse.id}"
+
+  policy = <<POLICY
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "s3VersioningMfaFalse-restrict-access-to-users-or-roles",
```

Comment:
**Accurics remediation**
 
**[HIGH]: S3 bucket accessible to all authenticated users.**
 
**Vulnerability**
S3 bucket Access is allowed to all AWS Account Users.
 
**Remediation**
S3 bucket Access should be limited to specific users and roles.

source: accurics-